saas/managers/metrics.py

```diff
@@ -60,20 +76,23 @@ def month_periods(nb_months=12, from_date=None, step_months=1):
                 month = 12
             else:
                 month = month % 12
-        last = datetime(day=1, month=month, year=year, tzinfo=utc)
+        last = datetime(day=1, month=month, year=year)
+        last = _handle_tz(last, tz_ob, orig_tz)
         dates.append(last)
     dates.reverse()
-    return dates
+    if convert_to_utc:
+        dates = [date.astimezone(utc) for date in dates]
 
+    return dates
 
 def aggregate_monthly(organization, account,
-                      from_date=None, orig='orig', dest='dest', **kwargs):
+                      from_date=None, tz=None, orig='orig', dest='dest', **kwargs):
     # pylint: disable=too-many-locals
     counts = []
     amounts = []
     # We want to be able to compare *last* to *from_date* and not get django
     # warnings because timezones are not specified.
-    dates = month_periods(13, from_date)
+    dates = month_periods(13, from_date, convert_to_utc=True, tz=tz)
```

Comment:
It seems best to remove the `convert_to_utc` boolean parameter and create a separate function, then rewrite this code as `dates = convert_to_utc(month_periods(13, from_date, tz=tz))`.